lib/WebService/Stripe.pm

```diff
@@ -18,6 +19,14 @@ has api_key => (
     required => 1,
 );
 
+has serializer => (
```

Comment:
FWIW - This should just override the default: `has '+serializer' => (default => sub { ... });`.